exercises/practice/leap/.approaches/ternary-operator/snippet.txt

```diff
@@ -0,0 +1,2 @@
+def leap_year(year):
+    return (not year % 400 if not year % 100 else not year % 4)
```

Comment:
Same comment here.  I think we should lose the parens.

Reply:
I like it better without the parens, too, but I wonder why the online formatter I linked to in the other comment put them on. I would gladly shuck them!

Reply:
See my lecture in the `Bob` PR.  😉   All this stuff is **_convention_**.  That site's _convention_ is to do that.  And if I were working in a code base that did that, I would also follow along (thank the stars I don't!).

Dunno if I missed the reference in PEP8 (_just did searches in there on both parens and returns and nope_) ... but I just don't think the `()` adds clarity, and it absolutely drives me nuts when used in a `return`, since it often looks at first glance like someone is trying to return a `tuple`.  Of course, using typehints would solve that 😉   But honestly, they're just visual noise that don't add to understanding, IMHO.

Reply:
Thanks, see my other comment.